cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/MultivariateAnomalyDetection.scala

```diff
@@ -232,161 +237,107 @@ trait MADBase extends HasOutputCol
 
   setDefault(timestampCol -> "timestamp")
 
-  val intermediateSaveDir = new Param[String](this, "intermediateSaveDir", "Directory name " +
-    "of which you want to save the intermediate data produced while training.")
+  private def validateIntermediateSaveDir(dir: String): Boolean = {
+    assert(dir.startsWith("wasbs://"), "improper HDFS loacation. Please use a wasb path such as: \n" +
```

Comment:
Is only wasbs accepted? What about abfss and say in Trident some other protocols?

Reply:
Added abfss too